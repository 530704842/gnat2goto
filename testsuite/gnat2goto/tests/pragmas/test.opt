ALL XFAIL CBMC fails with invariant check failure in boolbv_width.cpp
