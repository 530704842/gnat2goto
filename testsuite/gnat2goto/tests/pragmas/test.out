Standard_Error from gnat2goto pragmas:
pragmas.adb:2:04: warning: variable "i" is read but never assigned

[1] : SUCCESS
[2] : FAILURE
VERIFICATION FAILED
